Example/OriginateUI/TableViewController.m

```diff
@@ -0,0 +1,172 @@
+{
+    if (!_table){
+        self.table = [[UITableView alloc] initWithFrame:self.view.bounds style:UITableViewStylePlain];
+        self.table.backgroundColor = [UIColor whiteColor];
+        self.table.delegate = self;
+        self.table.dataSource = self;
+    }
+    return _table;
+}
+
+- (NSArray *)details
+{
+    if (!_details){
+        self.details = @[
+                         @"Specifiy insets for text rendering or other elements",
+                         @"For text fields that would benefit from validation.",
+                         @"For text fields that would benefit from validation.",
+                         @"Enables easy access to motion interpolation on views.",
+                         @"Two methods on UIImage to make tinting easier.",
+                         @"Allows use of hexadecimal colors with UIColor.",
+                         @"Easily create circular labeled images.",
+                         @"Install OriginateUI with just a few lines of code.",
+                         @"OriginateUI is available under the MIT license.",];
+    }
+    return _details;
+}
+
+- (NSArray *)labels
+{
+    if (!_labels) {
+        self.labels = @[ @"Text Fields", @"Validating Text Fields", @"Gradient Views", @"Motion Interpolation",
+                         @"Image Tinting", @"Hexadecimal Colors", @"Circle Images", @"Installation", @"Liscense",];
+    }
+    return _labels;
+}
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView
+{
+    return 1.0;
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section
+{
+    return self.labels.count;
+}
+
+- (void)tableView:(UITableView *)tableView willDisplayCell:(UITableViewCell *)cell forRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    cell.backgroundColor = [UIColor whiteColor];
+}
+
+- (CGFloat)tableView: (UITableView*)tableView heightForRowAtIndexPath: (NSIndexPath*) indexPath
+{
+    return (self.view.frame.size.height - 64.0)/9.0;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    static NSString *cellIdentifier = @"cell";
+    UITableViewCell *cell = [self.table dequeueReusableCellWithIdentifier:cellIdentifier];
+    if (cell == nil) {
+        cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleSubtitle reuseIdentifier:cellIdentifier];
+    }
+    cell.textLabel.text =  [self.labels objectAtIndex:indexPath.row];
+    cell.textLabel.font = [UIFont fontWithName:@"CircularPro-Book" size:17.0];
+    cell.detailTextLabel.text = [self.details objectAtIndex:indexPath.row];
+    cell.detailTextLabel.font = [UIFont fontWithName:@"MillerDisplay-Roman" size: 12.0];
+    cell.accessoryType = UITableViewCellAccessoryDisclosureIndicator;
+//    [cell setSeparatorsToFullWidth];
+    return cell;
+}
+
+- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    switch (indexPath.row) {
+        case 0:
```

Comment:
Suggest to enumerate the numbers, so it's cleaner to see what does the number represent :D 

For example, the enumeration would be something like
```
typedef NS_ENUM(NSInteger, ControllerType) {
    ControllerTypeTextField,
    ControllerTypeValidation,
    ControllerTypeGradient,
    ControllerTypeMotion,
    ControllerTypeTinting,
    ControllerTypeHexadecimal,
    ControllerTypeInstallation,
    ControllerTypeLicense
};
```

Then when u switch, just use
```
switch(indexPath.row) {
    case ControllerTypeTextField: {
        TextFieldViewController *textFieldViewController =  [[TextFieldViewController alloc] init];
        [self.navigationController pushViewController:textFieldViewController animated:YES];
    }
    break;
    ...
}
```